src/Atlabs/README.md

```diff
@@ -0,0 +1,385 @@
+# INNOVATION SANDBOX
+
+## Install
+The recommended way to install innovation-sandbox is through Composer.
+
+```bash
+$ composer require enyata/innovation-sandbox
+```
+
```

Comment:
Please sync with @addempsea regarding documentation standard in node package

Reply:
noted